ext/drcovlib/modules.c

```diff
@@ -459,10 +459,9 @@ module_table_entry_print(module_entry_t *entry, char *buf, size_t size)
 #endif
     read_entry.path = full_path;
     read_entry.custom = entry->custom;
-#ifndef WINDOWS
-    // For unices we record the physical offset from the backing file.
+    // For unices we record the physical offset from the backing file
+    // (always 0 on Windows).
```

Comment:
nit: maybe use /* */  instead for the comment.

Reply:
That's from snehasish's original code but it is a C file so sure I could tweak it to better match the style.